src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -1,6 +1,16 @@
 import {SpanEntry} from 'app/components/events/interfaces/spans/types';
 import {API_ACCESS_SCOPES} from 'app/constants';
 import {Field} from 'app/views/settings/components/forms/type';
+import {
+  INSTALLED,
+  NOT_INSTALLED,
+  PENDING,
+} from 'app/views/organizationIntegrations/constants';
+
+export type IntegrationInstallationStatus =
```

Comment:
Finally made a type for the installation status :)

Reply:
@scefali thanks for updating the other components with the new type!